docs/manually-deploying-atmosphere-ansible.md

```diff
@@ -155,15 +142,15 @@ The latest code (usually `master`) for Atmosphere Ansible.
 		60%
 		Networking  10.0.1.27   Tiny1   Atmo Cloud - USA
 		```
-		
+
 	1. Generate hosts
 
 		```
 		cat instances.txt | grep 10| awk '{ i++;printf "vm-%s ansible_host=%s ansible_port=22\n",i,$2}'
 		```
-		
-1. Run Ansible
+
+1. Run Ansible. from `<path-to-repo-clone>/ansible/`:
 
 	```
 	ansible-playbook all-the-pbs.yml -i hosts -e @deploy_vars.yml --limit "vm-*"
-	```
\ No newline at end of file
+	```
```

Comment:
This might be a moot point for the more Ansible-savvy - but you it _could_ be wrote adding or including that playbooks (without dependence on prior execution) can be run using a variation of this format:

```
# as noted, run from within `<path-to-repo-clone>/ansible/`
ansible-playbook playbooks/utils/check_networking.yml -i hosts -e @deploy_vars.yml --limit "vm-*"
```